compiler/noirc_frontend/src/elaborator/traits.rs

```diff
@@ -53,6 +59,21 @@ impl<'context> Elaborator<'context> {
         self.current_trait = None;
     }
 
+    fn resolve_trait_bounds(
+        &mut self,
+        unresolved_trait: &UnresolvedTrait,
+    ) -> Vec<ResolvedTraitBound> {
+        let mut resolved_trait_bounds = Vec::new();
+
+        for trait_bound in &unresolved_trait.trait_def.bounds {
+            if let Some(resolved_trait_bound) = self.resolve_trait_bound(trait_bound) {
+                resolved_trait_bounds.push(resolved_trait_bound);
+            }
+        }
+
+        resolved_trait_bounds
```

Comment:
```suggestion
        unresolved_trait.trait_def.bounds.iter()
            .filter_map(|bound| self.resolve_trait_bound(bound))
            .collect()
```

Reply:
Done, thanks! (I didn't apply this because rustmt formats it differently)